draft-ietf-rtcweb-jsep.xml

```diff
@@ -2542,13 +2542,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             parameters for that codec MUST be specified to indicate
             that silence suppression for received audio is not desired.
             For example, when using the Opus codec, the "usedtx=0"
-            parameter would be specified in the offer.</t>
-
-            <t>Note that setting the "VoiceActivityDetection" parameter
-            when generating an offer is a request to receive audio with
-            silence suppression. It has no impact on whether the local
-            endpoint does silence suppression for the audio it
-            sends.</t>
+            parameter would be specified in the offer. In addition,
+            the implementation MUST NOT use silence suppression for
+            media it generates, regardless of whether the "CN"
+            codecs or DTX fmtp parameters appear in the peer's
```

Comment:
We don't mention 'DTX' fmtp params anywhere else in this section. Perhaps 'DTX' -> 'related'